eradiate/kernel/transform.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any
+
+
+def map_unit_cube(
+    xmin: float, xmax: float, ymin: float, ymax: float, zmin: float, zmax: float
+) -> Any:
+    """Map the unit cube to [xmin, xmax] x [ymin, ymax] x [zmin, zmax].
```

Comment:
We try to start all docstrings with a newline (required for class docstrings, good practice for others):

```suggestion
    """
    Map the unit cube to [xmin, xmax] x [ymin, ymax] x [zmin, zmax].
```